.github/workflows/main.yml

```
@@ -15,7 +15,7 @@ jobs:
         uses: actions/checkout@v2
 
       - name: Use Node ${{ matrix.node }}
-        uses: actions/setup-node@v1
+        uses: actions/setup-node@v2.1.5
```

Using the major version will create less PRs

```suggestion
        uses: actions/setup-node@v2
```